src/Microsoft.Framework.PackageManager/Install/UninstallCommand.cs

```diff
@@ -78,6 +78,7 @@ private void PurgeOrphanPackages()
             foreach(var applicationPackage in applicationPackagesStillUsed)
             {
                 var appLockFileFullPath = Path.Combine(
+                    packagesRepo.RepositoryRoot.Root,
```

Comment:
`_commandsRepo.PackagesRoot` I think has the same value and might be a better choice